Lib/test/test_typing.py

```diff
@@ -3809,6 +3809,23 @@ class Y(C[int]):
         self.assertEqual(Y.__qualname__,
                          'GenericTests.test_repr_2.<locals>.Y')
 
+    def test_repr_3(self):
+        T = TypeVar('T')
+        P = ParamSpec('P')
+
+        class MyCallable(Generic[P, T]):
+            pass
+
+        object_to_expected_repr = {
+            MyCallable[[], bool]:         "MyCallable[[], bool]",
+            MyCallable[[int], bool]:      "MyCallable[[int], bool]",
+            MyCallable[[int, str], bool]: "MyCallable[[int, str], bool]"
+        }
+
+        for obj, expected_repr in object_to_expected_repr.items():
+            with self.subTest(obj=obj, expected_repr=expected_repr):
+                self.assertEqual(repr(obj), MyCallable.__module__ + expected_repr)
```

Comment:
Oh this test will fail now, I forgot the `.` in the middle 🤦

```suggestion
                self.assertEqual(repr(obj), f"{MyCallable.__module__}.{expected_repr}")
```